### PR TITLE
Fix: Some changes in quests.

### PR DESCRIPTION
**Pull request by henriquesameshima**
1. Removed "bitwise_flags.lua" and changed function calls to "bit.lua".
2. Changed quests that use "testFlag" replaced by "hasBitSet".
3. Fixed NPC The Empress that was not continuing a mission.

Co-Author: @RodrigoSilva93

### CODE REVIEW

**Comment by dudantas** [comment]
Can you change the indentation? For example, leave everything for space or tab

**Comment by henriquesameshima** [comment]
I changed the file that was with different indentation and changed everything to tabs.

**Comment by henriquesameshima** [comment]
1. Fixed a part of the Kilmaresh quest where it was possible to count dead monsters without being on the correct mission.
2. Fixed loot Starving Wolf
3. Fixed King Zelos correct corpse id.

**Review by majestyotbr — APPROVED** [review]

File: data/scripts/creaturescripts/quests/kilmaresh/fafnar.lua [hunk 1]
```diff
@@ -7,11 +7,11 @@ local fafnar = CreatureEvent("FafnarKill")
 
 function fafnar.onKill(creature, target)
 	local storage = monster[target:getName():lower()]
-	if target:isPlayer() or target:getMaster() or not storage then
+	if target:isPlayer() or target:getMaster() or not storage or creature:getStorageValue(Storage.Kilmaresh.Twelve.Boss) ~= 3 then
 		return false
 	end
 
-	local kills = creature:getStorageValue(storage)
+	local kills = creature:getStorageValue(storage) and 0
```
Comment (dudantas) [inline data/scripts/creaturescripts/quests/kilmaresh/fafnar.lua:14]: Why "and 0"?

File: data/scripts/quests/kilmaresh/1-fafnars-wrath/6-mind-travel.lua [hunk 2]
```diff
@@ -81,38 +81,37 @@ rightMirror:register()
...
+		player:setStorageValue(Storage.Kilmaresh.Fifth.Memories, memoryStorage + 1)
+		player:setStorageValue(Storage.Kilmaresh.Fifth.MemoriesShards, setFlag(memoriesShardsStorage, memoryShardsItemIdsBitmasks[item:getId()]))
+		toPosition:sendMagicEffect(CONST_ME_ENERGYAREA)
+		return true
+	end
+	return false
 end
 
-memoryShards:id(shardIds) -- Green, blue and purple memory shards
+memoryShards:id(36189, 36190, 36191) -- Green, blue and purple memory shards
```
Comment (dudantas) [inline data/scripts/quests/kilmaresh/1-fafnars-wrath/6-mind-travel.lua:114]: Why? Isn't it better to keep track of the table up there?

**Comment by dudantas** [comment]
Why the removal of bitwise flags? It’s already in the repo, from what I’ve seen, but they’re different.

**Comment by majestyotbr** [comment]
PR closed due to inactivity. If you want, you can reopen to finish.

**Pull request closed without merging**